tests/test_magma_protocol.py

```diff
@@ -0,0 +1,62 @@
+from typing import Optional
+import magma as m
+import fault
+from hwtypes import BitVector
+
+
+def test_foo_type_magma_protocol():
+    class FooMeta(m.MagmaProtocolMeta):
+        def _to_magma_(cls):
+            # Need way to retrieve underlying magma type
```

Comment:
can we kill these comments (they seem to be duplicated from the MagmaProtocolMeta definition)